### DISCUSSION TITLE
Error composer

### DISCUSSION THREAD

**Discussion by warsemann** [discoverlance-com/filament-page-hints#12] [Q&A]
Hi, i want to try your plugin but the install command retunr this error : 
`  Problem 1
    - Root composer.json requires discoverlance/filament-page-hints ^1.03, found discoverlance/filament-page-hints[dev-fix-modal-ui, dev-main, 0.0.1, 0.0.2, 0.0.3, 1.0.0, 1.0.1, 1.0.2, v1.0.3] but it does not match the constraint.`

**Comment by discoverlance-com** [comment]
> Hi, i want to try your plugin but the install command retunr this error : ` Problem 1 - Root composer.json requires discoverlance/filament-page-hints ^1.03, found discoverlance/filament-page-hints[dev-fix-modal-ui, dev-main, 0.0.1, 0.0.2, 0.0.3, 1.0.0, 1.0.1, 1.0.2, v1.0.3] but it does not match the constraint.`

What command did you use to install the package? Can you paste that here? And also your composer.json file?

Did you specify a certain version whilst installing or you used the normal installation command: `composer require discoverlance/filament-page-hints`? It looks like the version you specified is `1.03` but the package versions available if you want the latest, is `1.0.3` so it's missing a dot before the 3 ( `1.03 should rather be 1.0.3`)

**Comment by warsemann** [comment]
Hi, thanks to reply, when i run the normal installation command : 

`  Problem 1
    - Root composer.json requires stephenjude/filament-debugger ^1.2 -> satisfiable by stephenjude/filament-debugger[1.2.0].
    - discoverlance/filament-page-hints[0.0.1, ..., 0.0.3, 1.0.0, ..., v1.0.3] require illuminate/contracts ^9.0 -> satisfiable by illuminate/contracts[v9.0.0, ..., v9.52.4].
    - Conclusion: don't install illuminate/contracts v9.52.4 (conflict analysis result)
    - Root composer.json requires discoverlance/filament-page-hints * -> satisfiable by discoverlance/filament-page-hints[0.0.1, 0.0.2, 0.0.3, 1.0.0, 1.0.1, 1.0.2, v1.0.3].
`
My composer.json
`{
    "name": "laravel/laravel",
    "type": "project",
    "description": "The Laravel Framework.",
    "keywords": ["framework", "laravel"],
    "license": "MIT",
    "require": {
        "php": "^8.1",
        "alexjustesen/filament-spatie-laravel-activitylog": "^0.6.2",
        "alperenersoy/filament-export": "^0.2.8",
        "althinect/filament-spatie-roles-permissions": "^1.0",
        "barryvdh/laravel-dompdf": "^2.0",
        "filament/filament": "^2.0",
        "filament/spatie-laravel-media-library-plugin": "^2.0",
        "guzzlehttp/guzzle": "^7.2",
        "jeffgreco13/filament-breezy": "^1.5",
        "konnco/filament-import": "^1.4",
        "laravel/framework": "^10.0",
        "laravel/sanctum": "^3.2",
        "laravel/tinker": "^2.8",
        "livewire/livewire": "^2.12",
        "spatie/browsershot": "^3.57",
        "spatie/laravel-activitylog": "^4.7",
        "spatie/laravel-medialibrary": "^10.0.0",
        "spatie/laravel-permission": "^5.9",
        "stephenjude/filament-debugger": "^1.2"
    },
    "require-dev": {
        "barryvdh/laravel-ide-helper": "^2.13",
        "fakerphp/faker": "^1.9.1",
        "laravel/pint": "^1.0",
        "laravel/sail": "^1.18",
        "mockery/mockery": "^1.4.4",
        "nunomaduro/collision": "^7.0",
        "phpunit/phpunit": "^10.0",
        "spatie/laravel-ignition": "^2.0"
    },
    "autoload": {
        "psr-4": {
            "App\\": "app/",
            "Database\\Factories\\": "database/factories/",
            "Database\\Seeders\\": "database/seeders/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\": "tests/"
        }
    },
    "scripts": {
        "post-autoload-dump": [
            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
            "@php artisan package:discover --ansi"
        ],
        "post-update-cmd": [
            "@php artisan vendor:publish --tag=laravel-assets --ansi --force"
        ],
        "post-root-package-install": [
            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
        ],
        "post-create-project-cmd": [
            "@php artisan key:generate --ansi"
        ]
    },
    "extra": {
        "laravel": {
            "dont-discover": []
        }
    },
    "config": {
        "optimize-autoloader": true,
        "preferred-install": "dist",
        "sort-packages": true,
        "allow-plugins": {
            "pestphp/pest-plugin": true,
            "php-http/discovery": true
        }
    },
    "minimum-stability": "stable",
    "prefer-stable": true
}
`
